Makefile

```
@@ -41,7 +41,7 @@ GINKGO_VER := v1.16.2
 GINKGO_BIN := ginkgo
 GINKGO := $(TOOLS_BIN_DIR)/$(GINKGO_BIN)-$(GINKGO_VER)
 
-KIND_VER := v0.12.0
+KIND_VER := v0.13.0
```

Let's use the latest kind version

```suggestion
KIND_VER := v0.14.0
```